spiketools/tests/spatial/test_occupancy.py

```diff
@@ -1,12 +1,24 @@
 """Tests for spiketools.spatial.occupancy"""
 
 from spiketools.spatial.occupancy import *
+import numpy as np
 
 ###################################################################################################
 ###################################################################################################
 
-def test_compute_spatial_bin_edges():
-    pass
+def test_compute_spatial_bin_edges(position, bins):
+    x_edges, y_edges = compute_spatial_bin_edges(position, bins)
+	# dimension checks
```

Comment:
I think your spacing is a little off here, make sure the `#` lines are the same indentation as the code (4 spaces)